### PR TITLE
Add _pixelbuf support

### PR DESCRIPTION
**Pull request by rhooper**

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -18,6 +18,7 @@ ignore-patterns=
 #init-hook=
 
 # Use multiple processes to speed up Pylint.
+# jobs=1
```
Comment (siddacious) [inline .pylintrc]: Looks like there is a bunch of commented out changes; just make sure to remove them before you're done.

**Comment by siddacious** [comment]
@rhooper Since pypixelbuf and the API changes were both merged, is this no longer WIP and ready to merge? I took a look and it's good from what I can tell but it may be wise to have someone more familiar with the code. In the meanwhile I'll do some testing

**Comment by rhooper** [comment]
This is no longer a wip

**Comment by siddacious** [comment]
Before someone else merges, might be worth mentioning this issue in the NeoPixel repo that popped up:
https://github.com/adafruit/Adafruit_CircuitPython_NeoPixel/issues/65
I'm not sure how similar the DotStar implementation is to the NeoPixel one but perhaps DotStar has the same issue?

File: docs/conf.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
     'sphinx.ext.viewcode',
 ]
 
-# autodoc_mock_imports = ["digitalio", "busio"]
+autodoc_mock_imports = ["pypixelbuf"]
```
Comment (tannewt) [inline docs/conf.py]: Is this needed? Can't the real version be installed from pypi?

File: README.rst [hunk 3]
```diff
@@ -78,6 +78,50 @@ This example demonstrates the library with the single built-in DotStar on the
     pixels = adafruit_dotstar.DotStar(board.APA102_SCK, board.APA102_MOSI, 1)
     pixels[0] = (10, 0, 0)
 
+
+This example demonstrates the library with the DotStar Feather Wing and bounces Blinka.
+
+`Feather M4 Express <https://www.adafruit.com/product/3857>`_ and
+`DotStar FeatherWing <https://www.adafruit.com/product/3449>`_.
+
+.. code-block:: python
```
Comment (tannewt) [inline README.rst]: I think this example would be better to have in a separate example file. The example in the readme should be short and simple. Other examples can do fancier things.

**Pull request closed without merging**